helios-services/src/main/java/com/spotify/helios/servicescommon/coordination/Paths.java

```diff
@@ -29,6 +29,7 @@
   private static final String JOBS = "jobs";
   private static final String JOBREFS = "jobrefs";
   private static final String HOSTS = "hosts";
+  private static final String REMOVED_HOSTS = "removed";
```

Comment:
(Which will make it the only commented path amidst an entire file of paths that could really use explanatory comments.)